crowbar_framework/config/application.rb

```diff
@@ -42,5 +42,20 @@ class Application < Rails::Application
       g.test_framework :rspec, fixture: true
       g.fallbacks[:rspec] = :test_unit
     end
+
+    config.before_configuration do
+      Dotenv.load *Dir.glob(Rails.root.join("config", "*.env"))
```

Comment:
Lint/AmbiguousOperator: Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the `*` if it should be a multiplication. (https://github.com/bbatsov/ruby-style-guide#parens-as-args)